docs/modules/Asserts.md

```diff
@@ -396,7 +396,9 @@ Checks that condition is positive.
 
 
 ### expectException
- 
+
+Note: Deprecated. Use expectThrowable()
+
```

Comment:
This file is autogenerated from code, so your change will be lost.
The right place to make the change is https://github.com/Codeception/Codeception/blob/3.1/src/Codeception/Module/Asserts.php#L107

This method had `@deprecated` tag when `expectThrowable` was introduced, but it disappeared later: https://github.com/Codeception/Codeception/pull/5213/files#diff-b3d3622a11b4688b45ee6999ce435bc0

It looks like I tried to remove this method completely, but had to put it back for some reason and `@deprecated` line was lost in the process
https://github.com/Codeception/Codeception/commit/7511fd2ceabe071cb65c0f38ae459c2d392ebefa#diff-b3d3622a11b4688b45ee6999ce435bc0